pymc3/sampling.py

```diff
@@ -573,7 +573,15 @@ def init_nuts(init='ADVI', njobs=1, n_init=500000, model=None,
             progressbar=progressbar
         )  # type: pm.MeanField
         start = approx.sample(draws=njobs)
-        cov = approx.cov.eval()
```

Comment:
thanks! I will go with this. 
If I understand correctly the posterior cov of `advi` is a diagonal matrix right?